lib/interpreters/periodic_visits.rb

```diff
@@ -118,157 +114,113 @@ def generate_relations(vrp)
       }.compact.flatten
     end
 
+    def generate_relations_between_visits(vrp, mission)
+      # TODO : need to uniformize generated relations whether mission has minimum AND maximum lapse or only one of them
+      return unless mission.visits_number > 1
+
+      if mission.minimum_lapse && mission.maximum_lapse
```

Comment:
could you factorise the relation definition ? and give the specific mission_ids in a single place. It is a bit difficult to see the distinction between the cases.

Reply:
For now the goal was to group code that was identical for shipment and services.

I would indeed like to factorise but it is not that easy because of https://gitlab.com/mapotempo/optimizer-api/-/issues/339 . There can be an impact we need to test.